### PR TITLE
Add pages to the ZIM and display them in the UI

### LINKED ISSUE #7: Display page content in the ZIM UI
Once page tree is retrieved, we need to move forward and:
- retrieve page content with the API
- store this content in the ZIM
- display this content in the Vue.JS UI

Nota: this issue is not "functional", i.e. links will still be broken, CSS not properly applied, MathJax missing, images missing, Youtube video not working, this has been split is dedicated issues for simplicity of code review / progress monitoring

### PR DESCRIPTION
**Pull request by benoit74**
Fix #7 

Changes:
- fetch page content, store in the ZIM, display it
- remove `home.json` which is not needed anymore, home is just a normal page indeed
- added root page path to `shared.json`
- added summary page data to `shared.json` so that we can transform nice looking paths into page id and title
- modified integration test to fetch only a small subtree of the whole website inside the ZIM for quick execution

### CODE REVIEW

File: zimui/src/stores/main.ts [hunk 1]
```diff
@@ -39,6 +47,26 @@ export const useMainStore = defineStore('main', {
         }
       )
     },
+    async fetchPageContent(page: SharedPage) {
+      this.isLoading = true
+      this.errorMessage = ''
+      this.errorDetails = ''
+
+      return axios.get('./content/page_content_' + page.id + '.json').then(
```
Comment (rgaudin) [inline zimui/src/stores/main.ts:55]:
```suggestion
      return axios.get(`./content/page_content_${page.id}.json`).then(
```

File: zimui/src/stores/main.ts [hunk 2]
```diff
@@ -39,6 +47,26 @@ export const useMainStore = defineStore('main', {
...
+        (response) => {
+          this.isLoading = false
+          this.pageContent = response.data as PageContent
+        },
+        (error) => {
+          this.isLoading = false
+          this.shared = null
+          this.errorMessage = 'Failed to load page content for page ' + page.id
```
Comment (rgaudin) [inline zimui/src/stores/main.ts:63]:
```suggestion
          this.errorMessage = `Failed to load page content for page ${page.id}`
```

**Review by rgaudin — APPROVED** [review]
LGTM

**Comment by benoit74** [comment]
Thank you, still not used to this notation, I keep forgetting about it!

**Pull request merged**